### PR TITLE
Set up separate directories for hello-world app and call-me-back app

### PR DESCRIPTION
**Pull request by mscharfstein**
Move hello-world app into hello-world directory and copy the scaffolding over to a new call-me-back directory. Update call-me-back React frontend to be essentially blank, use the same backend as hello-world.

Thinking some of these bin/* files should not need to be pushed? Any thoughts? I noticed we have multiple gitignores so maybe we should combine those?

Changes besides just moving files:
   - Change the main/java/org/google/newprojectny directory to main/java/org/google/helloworld (and main/java/org/google/callmeback) and update the imports in the backend files as necessary
   - Adjusted "app" name in mongodb.yaml and web.yaml to "hello-world" instead of "new-project-ny" instructions within (and "call-me-back" in callmeback directory)
   - Adjusted ReadME instructions for starting docker to name the docker instances accordingly

### CODE REVIEW

File: call-me-back-app/web/.classpath [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mscharfstein) [inline call-me-back-app/web/.classpath:1]: is this file necessary? should it be ignored in git?

File: call-me-back-app/web/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mscharfstein) [inline call-me-back-app/web/.project:1]: same with this... can i remove it?

File: call-me-back-app/kubernetes/web.yaml [hunk 3]
```diff
@@ -3,12 +3,12 @@ kind: Service
 metadata:
   name: npny-web-service
```
Comment (tariqajyusuf) [inline call-me-back-app/kubernetes/web.yaml:4]: nit: Could you also update `npny` here to `cmb`? I kind of "namespaced" things and it would be great to keep a consistent convention

File: call-me-back-app/web/.classpath (hunk 1 above)
Comment (tariqajyusuf) [reply call-me-back-app/web/.classpath:1]: Did a bit of Googling and it seems this is an Eclipse thing, we can go ahead and add this to the `.gitignore` file at the top level.

File: call-me-back-app/web/.project (hunk 2 above)
Comment (tariqajyusuf) [reply call-me-back-app/web/.project:1]: Ditto.

File: call-me-back-app/web/.settings/org.eclipse.buildship.core.prefs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+arguments=
```
Comment (tariqajyusuf) [inline call-me-back-app/web/.settings/org.eclipse.buildship.core.prefs:1]: We can probably ignore the entire `.settings` directory

See https://github.com/gradle/gradle/blob/master/.gitignore#L39

File: hello-world-app/web/.project [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tariqajyusuf) [inline hello-world-app/web/.project:1]: We can also ignore `.project`

See: https://github.com/gradle/gradle/blob/master/.gitignore#L39

File: call-me-back-app/web/.project (hunk 2 above)
Comment (mscharfstein) [reply call-me-back-app/web/.project:1]: Done

File: call-me-back-app/web/.classpath (hunk 1 above)
Comment (mscharfstein) [reply call-me-back-app/web/.classpath:1]: Done

File: hello-world-app/web/.project (hunk 5 above)
Comment (mscharfstein) [reply hello-world-app/web/.project:1]: Done

File: call-me-back-app/web/.settings/org.eclipse.buildship.core.prefs (hunk 4 above)
Comment (mscharfstein) [reply call-me-back-app/web/.settings/org.eclipse.buildship.core.prefs:1]: Done

File: call-me-back-app/kubernetes/web.yaml (hunk 3 above)
Comment (mscharfstein) [reply call-me-back-app/kubernetes/web.yaml:4]: Done

**Comment by mscharfstein** [comment]
Thanks Tariq and Kurt! One comment below:

 "Add `.DS_Store` to the top level `.gitignore` so those files are not committed"
I added **/.DS_Store to get this file in any directory. Is that sufficient?

**Comment by egracer** [comment]
> Thanks Tariq and Kurt! One comment below:
> 
> "Add `.DS_Store` to the top level `.gitignore` so those files are not committed"
> I added **/.DS_Store to get this file in any directory. Is that sufficient?

I believe so, yes. You can check this by committing the .gitignore file, and then adding .DS_Store files in a few different subdirectories to see if they are included when you run `git status`.

**Comment by mscharfstein** [comment]
Done!
